src/modules/powerrename/lib/PowerRenameManager.cpp

```diff
@@ -1069,7 +1093,7 @@ DWORD WINAPI CPowerRenameManager::s_regexWorkerThread(_In_ void* pv)
                     wchar_t transformedName[MAX_PATH] = { 0 };
                     if (newNameToUse != nullptr && (flags & Uppercase || flags & Lowercase || flags & Titlecase || flags & Capitalized))
                     {
-                        winrt::check_hresult(GetTransformedFileName(transformedName, ARRAYSIZE(transformedName), newNameToUse, flags));
+                        winrt::check_hresult(GetTransformedFileName(transformedName, ARRAYSIZE(transformedName), newNameToUse, flags, isFolder));
```

Comment:
can we add try/catch block around `GetTransformedFileName` call just to be safe?

Reply:
added. if something is thrown, newName will fallback to empty `transformedName` string. so no crash will happen